core/src/components/cat-alert/cat-alert.scss

```diff
@@ -33,11 +38,9 @@
 @mixin theme($theme) {
   .alert-#{$theme} {
     --bg: #{cat-token('color.theme.#{$theme}.bg', $wrap: false)};
-    --text: #{cat-token('color.theme.#{$theme}.text', $wrap: false)};
-    @if $theme == 'secondary' {
-      --border: #{cat-token('color.theme.#{$theme}.bg', $wrap: false)};
-    } @else {
-      --border: #{cat-token('color.theme.#{$theme}.text', $wrap: false)};
+
+    cat-icon::part(icon) {
+      color: cat-token('color.theme.#{$theme}.bg');
```

Comment:
This should not be theme.bg but theme.text, I think

Reply:
certainly, but if i used text it wouldn't match with the design, so i added a new token to make it more readable